firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigAutoFetch.java

```diff
@@ -121,19 +119,26 @@ private void handleNotifications(InputStream inputStream) throws IOException {
     BufferedReader reader = new BufferedReader((new InputStreamReader(inputStream, "utf-8")));
     String message;
     while ((message = reader.readLine()) != null) {
+      if (!message.contains("latestTemplateVersionNumber")) {
+        continue;
+      }
+
       long targetTemplateVersion = configFetchHandler.getTemplateVersionNumber();
       try {
-        JSONObject jsonObject = new JSONObject(message);
+        JSONObject jsonObject = new JSONObject("{" + message + "}");
```

Comment:
Could it happen that message does include the `{}`?

Reply:
No, since we're using chunked encoding every line of the response is sent separately. So that's why we need to add the curly brackets; the brackets will be sent a the beginning and end of every full message.